about.html

```diff
@@ -0,0 +1,196 @@
+                            </li>
+                        </ul>
+                    </div>
+                </nav>
+            </div>
+
+    </header>
+    <main>
+        <section class="about-section-header">
+            <h2>ACCRA EDU-FAIR 2022</h2>
+            <span class="underline"></span>
+            <div class="about-section-para">
+                <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Obcaecati natus aspernatur alias quisquam architecto unde repudiandae ullam! Cum, molestiae commodi dignissimos aut veritatis dolorem obcaecati nam at non reprehenderit ipsum.</p>
+            </div>
+            <p class="about-section-para2">For enquiries and reservations please contact Lorem ipsum dolor sit amet.
+               <br>
+               "accraedusummit@eventbright.com"
+            </p>
+        </section>
+        <section class="about-section-logo">
+            <h3>Accra Edu-Fair 2022 Logo</h3>
+            <span class="underline"></span>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Veniam, explicabo. Cupiditate ea minima blanditiis ipsum expedita reiciendis id ut alias officiis. Ipsa, culpa cum! Dicta repudiandae magni obcaecati inventore dolores modi molestiae excepturi. Eveniet similique nobis voluptates nemo corporis alias. Omnis quis suscipit nisi et aliquid a repellendus explicabo obcaecati deserunt eligendi. Asperiores aperiam qui distinctio, rerum magni velit quaerat deleniti atque? Necessitatibus sequi at in repudiandae et. Earum ut iusto eveniet!</p>
+            <div class="about-section-logo-border">
+                <img src="./images/social.jpg.jpg" alt="" class="logo">
+                <img src="./images/social.jpg.jpg" alt="" class="logo">
+                <img src="./images/social.jpg.jpg" alt="" class="logo">
+            </div>
+        </section>
+        <section class="previous-events">
+            <h3>See the past Edu-Fair Events</h3>
+            <span class="underline"></span>
+            <p>Take a look at the last two Edu-Fair events which took place in Kumasi and Takoradi</p>
+            <div class="edu-events">
+                <div class="event1">
+                    <p>2020 Edu-Fair Event
+                        <br>
+                        Kumasi
+                    </p>
+                    <img src="./images/event1.jpg" alt="picture of past event" class="edu-fair-2020">
+                </div>
+                <div class="event2">
+                    <p>2018 Edu-Fair Event
+                        <br>
+                        Takoradi
+                    </p>
+                    <img src="./images/event2.jpg" alt="picture of previous event" class="edu-fair-2020">
+                </div>
+                <div class="event2"></div>
+            </div>
+        </section>
+        <section class="sponsors">
+            <h3>Sponsors</h3>
+            <span class="underline"></span>
+            <div class="sponsors-logo">
```

Comment:
- [ ] [Optional] please, check the alignments, padding, width, and height of the images. Disorganizing and unaligned images make your website looks ugly and unclean.